packages/phone-number-privacy/signer/test/index.test.ts

```diff
@@ -66,6 +66,7 @@ describe(`POST /getBlindedMessageSignature endpoint`, () => {
           {
             success: true,
             signature: BLS_SIGNATURE,
+            version: VERSION,
```

Comment:
If you want to return this from the signer, you'll have to update `SignMessageResponse` interface that the combiner defines as the return object to include `version` field as well.

Reply:
Missed this, thank you!